src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double firstSide;
+    private double secondSide;
+
+    public Rectangle(Color color, double firstSide, double secondSide) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double calculateArea() {
+        return firstSide * secondSide;
+    }
+
+    @Override
+    public void drawFigure() {
+        System.out.println("Figure: rectangle, area: " + calculateArea()
+                + " sq. units, firstSide: " + getFirstSide() + " units, secondSide "
+                + getSecondSide() + " units, color: " + getColor());
```

Comment:
Instead of concatenating strings in the `drawFigure` method, consider using `String.format` for better readability.